packages/flutter/test/foundation/profile_test.dart

```diff
@@ -0,0 +1,18 @@
+// Copyright 2017 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/foundation.dart';
+import 'package:test/test.dart';
+
+const bool isReleaseMode = const bool.fromEnvironment("dart.vm.product");
```

Comment:
fwiw, now that i think of this some more: this will always be false. We run all our tests in debug mode always.